examples/tfengine/generated/full/live/prod/team1/example-prod-data/resources/variables.tf

```diff
@@ -1,19 +0,0 @@
-# Copyright 2020 Google LLC
```

Comment:
why does this change delete generated files for full?

Reply:
Good catch!! I did a bad copy of full.hcl from a bad branch which caused the deletions. I fixed it now. Thanks